Fabric.Identity.SqlServer/Admin/MountPoints.sql

```diff
@@ -0,0 +1,20 @@
+﻿/*
+Do not change the database path or name variables.
+Any sqlcmd variables will be properly substituted during 
+build and deployment.
+*/
+
+ALTER DATABASE [$(DatabaseName)]
+    ADD FILE (NAME = [HCFabricIdentityData1File1], FILENAME = '$(FabricIdentityDataMountPoint)\HC$(DatabaseName)Data1File1.ndf', SIZE = 100 MB) TO FILEGROUP [HCFabricIdentityData1];
```

Comment:
Let's check with some of our other teams to see if we should add a MAXSIZE and FILEGROWTH attributes here.

Reply:
Do we also need an index file definition?